SubmittedAssignments/Sanjeevanee_balure_03_/index.html

```
@@ -0,0 +1,809 @@
+    <div class="container">
+      <div class="row">
+        <!-- Restaurant Interior -->
+        <div class="col-md-6 text-center mb-4">
+          <img src="C:\Users\Sanjeevanee\Desktop\img.png" alt="Restaurant Interior" class="img-fluid rounded shadow">
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <!-- Section -->
+  <Section class="section bg-dark text-light py-3">
```

HTMLHint (1.5.0)

[error] The html element name of [ Section ] must be in lowercase.

(tagname-lowercase)